libtextsecure/message_receiver.js

```diff
@@ -1904,43 +1951,12 @@ MessageReceiver.prototype.extend({
   },
   async handleEndSession(number) {
     window.log.info('got end session');
-    const deviceIds = await textsecure.storage.protocol.getDeviceIds(number);
-    const identityKey = StringView.hexToArrayBuffer(number);
     let conversation;
     try {
```

Comment:
this can now be simplified to
```js
try {
    const conversation = window.ConversationController.get(number);
    if (conversation) {
        await conversation. onSessionResetReceived();
    } catch (e) {
        // log error here
    }
}
```